packages/pixeloven-webpack/file-reporter/src/helper.ts

```diff
@@ -0,0 +1,185 @@
+            }
+            resolve({
+                root,
+                sizes,
+            });
+        });
+    });
+}
+
+const friendlySyntaxErrorLabel = "Syntax error:";
+
+function isLikelyASyntaxError(message: string) {
+    return message.indexOf(friendlySyntaxErrorLabel) !== -1;
+}
+
+/**
+ * @todo clean these rules up and make them more relevant to our env.
+ *      This is a port of https://github.com/facebook/create-react-app/blob/master/packages/react-dev-utils/formatWebpackMessages.js
+ * @param message
+ */
+function formatMessage(message: string) {
+    let lines = message.split("\n");
+
+    // Strip Webpack-added headers off errors/warnings
+    // https://github.com/webpack/webpack/blob/master/lib/ModuleError.js
+    lines = lines.filter(line => !/Module [A-z ]+\(from/.test(line));
+
+    // Transform parsing error into syntax error
+    // TODO: move this to our ESLint formatter?
+    lines = lines.map(line => {
+        const parsingError = /Line (\d+):(?:(\d+):)?\s*Parsing error: (.+)$/.exec(
+            line,
+        );
+        if (!parsingError) {
+            return line;
+        }
+        const [, errorLine, errorColumn, errorMessage] = parsingError;
+        return `${friendlySyntaxErrorLabel} ${errorMessage} (${errorLine}:${errorColumn})`;
+    });
+
+    message = lines.join("\n");
+    // Smoosh syntax errors (commonly found in CSS)
+    message = message.replace(
+        /SyntaxError\s+\((\d+):(\d+)\)\s*(.+?)\n/g,
+        `${friendlySyntaxErrorLabel} $3 ($1:$2)\n`,
+    );
+    // Clean up export errors
+    message = message.replace(
+        /^.*export '(.+?)' was not found in '(.+?)'.*$/gm,
+        `Attempted import error: '$1' is not exported from '$2'.`,
+    );
+    message = message.replace(
+        /^.*export 'default' \(imported as '(.+?)'\) was not found in '(.+?)'.*$/gm,
+        `Attempted import error: '$2' does not contain a default export (imported as '$1').`,
+    );
+    message = message.replace(
+        /^.*export '(.+?)' \(imported as '(.+?)'\) was not found in '(.+?)'.*$/gm,
+        `Attempted import error: '$1' is not exported from '$3' (imported as '$2').`,
+    );
+    lines = message.split("\n");
+
+    // Remove leading newline
+    if (lines.length > 2 && lines[1].trim() === "") {
+        lines.splice(1, 1);
+    }
+    // Clean up file name
+    lines[0] = lines[0].replace(/^(.*) \d+:\d+-\d+$/, "$1");
+
+    // Cleans up verbose "module not found" messages for files and packages.
+    if (lines[1] && lines[1].indexOf("Module not found: ") === 0) {
+        lines = [
+            lines[0],
+            lines[1]
+                .replace("Error: ", "")
+                .replace(
+                    "Module not found: Cannot find file:",
+                    "Cannot find file:",
+                ),
+        ];
+    }
+
+    // Add helpful message for users trying to use Sass for the first time
+    if (lines[1] && lines[1].match(/Cannot find module.+node-sass/)) {
```

Comment:
Should this be `dart-sass` now?

Reply:
Probably just do both but again this is a copy paste and honestly we probably don't need it. I want to clean this all up but not right now.